src/handbook/customer/sales/engagements.md

```diff
@@ -26,17 +26,15 @@ countersignature.
 
 ### Steps
 
-1. Provide access to the bought resources
+1. [Start invoice procedure](/handbook/operations/billing/#creating-an-invoice)
+1. Send an email introducing the Customer Success Manager and providing access to the bought resources
```

Comment:
So the CSM provides access to the resources, and generates the license?

Reply:
No I still do that, it's just more natural to send the customer 1 welcome email with everything, intro to Rob, license key/have them create cloud account, etc. I'm updating the handbook to reflect how I actually onboard.

Reply:
I was confused why it's on one numbered item, but trying to split it made it clear what the issue is. Give me a few minutes to think about it